### DISCUSSION TITLE
NIM Installation on OpenShift Cluster

### DISCUSSION THREAD

**Discussion by sachsachdeva** [NVIDIA/nv-ingest#170] [Q&A]
Hello, I see most of the NIM Examples are around Docker and recently could find few ones on Minikube but very limited. We have decided to run it on OpenShift Cluster but finding it too tedious to first change the docker compose to Kubernetes equivalent and then running through multiple issues to make it work on OpenShift. Just thinking if any one else faced the similar issue. Also, would appreciate if you have any guidelines here. Thanks!

**Comment by jarmak-nv** [comment]
@jdye64 - any thoughts here?

**Comment by randerzander** [comment]
We have [Helm charts](https://github.com/NVIDIA/nv-ingest/tree/main/helm) for deploying to kubernetes.

I'm not familiar with OpenShift, but a quick glance [here](https://www.redhat.com/en/technologies/cloud-computing/openshift/helm) says it supports Helm charts. Let us know if this works, @sachsachdeva

**Reply by jdye64** [reply]
The Helm charts should work on OpenShift but we have not explicitly tried it. If you run into issues let us know and we can try and sort things out.
